src/RubberDuckyParking/parking_api/models.py

```diff
@@ -1,6 +1,6 @@
 from django.db import models
 from django import forms
-from django.contrib.auth.models import User
+from django.contrib.auth.models import AbstractUser
```

Comment:
what are the default fields for a abstractUser?

Reply:
It's the same as a User, but it's for inheritance